bin/collector-redis

```diff
@@ -1,2 +0,0 @@
-#!/usr/bin/env sh
-bin/sbt 'project zipkin-collector-service' 'run -f zipkin-collector-service/config/collector-redis.scala'
```

Comment:
`bin/collector-redis` has the same character count as `./bin/collector redis` :p